libmdnsd/CMakeLists.txt

```
@@ -63,37 +68,35 @@ if (MDNSD_BUILD_FUZZING_CORPUS)
 endif()
 
 if(MDNSD_COMPILE_AS_CXX)
-  set_source_files_properties(${LIBRARY_FILES} PROPERTIES LANGUAGE CXX)
+    set_source_files_properties(${LIBRARY_FILES} PROPERTIES LANGUAGE CXX)
 endif()
 
-if(MDNSD_BUILD_OSS_FUZZ)
-    add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
-else()
-    add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
-endif()
+add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
+set_property(TARGET libmdnsd PROPERTY OUTPUT_NAME "mdnsd")
```

```suggestion
set_property(TARGET libmdnsd PROPERTY OUTPUT_NAME "mdnsd")
set_property(TARGET libmdnsd PROPERTY DEFINE_SYMBOL "MDNSD_DYNAMIC_LINKING_EXPORT")
```
See https://cmake.org/cmake/help/latest/prop_tgt/DEFINE_SYMBOL.html